src/types.ts

```diff
@@ -98,8 +93,9 @@ export enum FundRaiseType {
 }
 
 export enum Features {
-  CustomModulesAllowed = 'CustomModulesAllowed',
-  FreezeMintingAllowed = 'FreezeMintingAllowed',
+  CustomModulesAllowed = 'customModulesAllowed',
+  FreezeMintingAllowed = 'freezeMintingAllowed',
+  DisableControllerAllowed = 'disableControllerAllowed',
```

Comment:
It shouldn't be here

Reply:
wich one?

Reply:
`DisableControllerAllowed = 'disableControllerAllowed',`